components/brave_wallet/browser/keyring_service.cc

```diff
@@ -413,7 +422,12 @@ std::string KeyringService::GetAccountAddressForKeyring(
 }
 
 // static
-std::string KeyringService::GetAccountPathByIndex(size_t index) {
+std::string KeyringService::GetAccountPathByIndex(
+    size_t index,
+    const std::string& keyring_id) {
+  if (keyring_id == mojom::kFilecoinKeyringId) {
+    return base::NumberToString(index);
+  }
```

Comment:
This is not right. Its derivation path should be `m/44'/461'/0'/0/index`
https://github.com/glifio/wallet/blob/3a36db74ce2d2ea55add1867d47ee1a13d75e3d7/utils/createPath/index.js#L6
you are currently using ETH's derivation path for FIL
`keyring->ConstructRootHDKey(*seed, kRootPath);` which is `m/44'/60'/0'/0` 
so keys you derived is not right, you can compare the addresses with filecoin wallet using the same mnemonics
they will be different

Reply:
done, thanks for checking